source/classes.tex

```diff
@@ -3306,10 +3306,10 @@
 \indextext{\idxcode{union}!global anonymous}%
 \indextext{scope!anonymous \tcode{union} at namespace}%
 Anonymous unions declared in a named namespace or in the global
-namespace shall be declared \tcode{static}. Anonymous unions declared at
-block scope shall be declared with any storage class allowed for a
-block-scope variable, or with no storage class. A storage class is not
-allowed in a declaration of an anonymous union in a class scope.
+namespace shall be declared \tcode{static}. A block-scope anonymous union declaration
+shall not contain a \grammarterm{storage-class-specifier} that is not permitted
+in the declaration of a block-scope variable. A \grammarterm{storage-class-specifier}
+shall not appear in a class scope anonymous union declaration.
```

Comment:
Please hyphenate class-scope when used as an adjective.

Because you're significantly revising this paragraph, please change the affected lines to use semantic linebreaks (see https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#latex-source-code-formatting).